cloud/scope/machine.go

```diff
@@ -19,6 +19,7 @@ package scope
 import (
 	"context"
 	"encoding/base64"
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
we're importing k8s.io/utils/pointer as well on L30, choose one perhaps?

Reply:
true let me fix that

Reply:
I kept autorest/to because pointers doesn't have pointer to string